MAUI/MauiAppWithBroker/MSALClient/PCAWrapper.cs

```diff
@@ -30,16 +38,30 @@ public class PCAWrapper
         // private constructor for singleton
         private PCAWrapper()

```

Comment:
Is the name `PCAWrapper` intuitive ? I know what it means because I work with PCA / CCA every day, but not sure about our users.

Reply:
Agreed.

`PublicClientWrapper`?